src/mongo/integration/standalone/dbclient_test.cpp

```diff
@@ -1414,7 +1414,11 @@ namespace {
         if (serverGTE(&c, 2, 4) && (!serverGTE(&c, 2, 7) || kCompiledWithSSL)) {
             createUser(c, TEST_DB, "user3", "password3");
             std::string errmsg;
-            ASSERT_FALSE(c.auth("test", "user3", "notPassword3", errmsg));
+            try {
+                ASSERT_FALSE(c.auth("test", "user3", "notPassword3", errmsg));
+            } catch (const DBException&) {
+                //Expected on 2.2
```

Comment:
I think it might even be worth providing a std::pair<int, int> serverVersion() method on DBClientBase itself (with a properly bikeshedded return value), but at this point in the release cycle it seems better to leave as is.